src/shared/buttons/SubmitButton.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
 import { SubmitButtonType } from './types';
 
-function SubmitButton({ onClick, title = 'Add' }: SubmitButtonType) {
+function SubmitButton({ onClick, title = 'Add', className = 'inline-flex items-center gap-2 bg-transit-green-dark px-4 py-2 rounded text-transit-white' }: SubmitButtonType) {
   return (
     <button
       type="button"
       onClick={onClick}
-      className="inline-flex items-center gap-2 bg-transit-green-dark px-4 py-2 rounded text-transit-white"
+      className={className}
```

Comment:
I suggest to change to
`className={classNames(style, className)}`
and above add `const style = <style which will be the same for every input> `(in this case: `"flex justify-center items-center bg-transit-green-dark rounded text-transit-white w-full h-full'`)
In this case if will be needed more style for some exception we will add it in props `className`